# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/apz/util/APZEventState.cpp
# Commit: 37f6acd499b1
# Full Hash: 37f6acd499b11386a9c97f5783c89c4908692029
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2025-05-16 16:19:07
# Regressor Bug: 1550462
# File Overlap Count: 1
# Description:
#   Bug 1550462 - part 2: Make PresShell::HandleEvent dispatch preceding pointerrawupdate event r=smaug,dom-core,edgar
#   
#   This patch tries to dispatch ePointerRawUpdate with
#   PresShell::EventHandler::DispatchPrecedingPointerEvent as same as usual
#   pointer events.
# ==============================================================================

diff -r 32ba02fa32ec -r 37f6acd499b1 gfx/layers/apz/util/APZEventState.cpp
--- a/gfx/layers/apz/util/APZEventState.cpp	Fri May 16 09:49:43 2025 +0000
+++ b/gfx/layers/apz/util/APZEventState.cpp	Fri May 16 09:49:44 2025 +0000
@@ -336,6 +336,9 @@
       break;
     }
 
+    case eTouchRawUpdate:
+      break;
+
     default:
       MOZ_ASSERT_UNREACHABLE("Unknown touch event type");
       break;